scripts/liveValidation.js

```diff
@@ -68,7 +68,7 @@ async function runScript() {
     let validationId = JSON.parse(response).validationId;
 
     let validationResultUrl = `${validationService}/${validationId}`;
-    console.log(`Request done, results will be available at ${validationResultUrl} in ${durationInSeconds} seconds...`);
+    console.log(`Request done, results will in ${durationInSeconds} seconds...`);
```

Comment:
"will be ready in" ?
I think it's good to remove it from here, if the output is useful for debugging (particularly for you or reviewers) feel free to write out the link, just somewhere else maybe in parenthesis or something - like "Note: raw results at <link>"

Reply:
well actually I print later everything that comes from there, but maybe ill add it as you mentioned "with raw results at"